statsd/statsd.go

```diff
@@ -23,32 +23,28 @@ var (
 	stats *statsDClient
 )
 
-func InitStatsD() {
-	host := Config.Get("statsd.host", "localhost")
-	port := Config.GetInt("statsd.port", 8125)
-	prefix := Config.Get("statsd.prefix", "my_service")
-
+func InitStatsD(host string, port int, prefix string) {
 	stats = initClient(fmt.Sprintf("%s:%d", host, port), prefix)
 }
 
 // StatsDMiddleware is the middleware handler that sends the status code and response time to StatsD server
-func StatsDMiddleware() gin.HandlerFunc {
+// func StatsDMiddleware() gin.HandlerFunc {
 
-	return func(c *gin.Context) {
-		startTime := time.Now()
-		c.Next()
-		responseTime := time.Since(startTime)
-		status_code := string(c.Writer.Status())
+// 	return func(c *gin.Context) {
+// 		startTime := time.Now()
+// 		c.Next()
+// 		responseTime := time.Since(startTime)
+// 		status_code := string(c.Writer.Status())
 
-		//Send metrics
-		go sendMetrics(status_code, "", responseTime)
+// 		//Send metrics
+// 		go SendMetrics(status_code, "", responseTime)
 
-	}
-}
+// 	}
+// }
 
 // StatsDWrapper wraps the worker work to it can measure its times and other stats.
 // it will return a func so we can continue and wrap it with other wrappers, such as logger.
-func StatsDWrapper(o options, worker wrappedFn) wrappedFn {
+func StatsDWrapper(o logger.Options, worker logger.WrappedFn) logger.WrappedFn {
```

Comment:
maybe it's good to seperate it from logger ? it was easy on shared. here it has no meaning.